src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import { Todo } from '../../types';
+import { TodoInfo } from '../TodoInfo';
+import './TodoList.scss';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer